src/ui-shell/header/header-action.component.ts

```diff
@@ -16,7 +16,6 @@ import {
 			[ngClass]="{
 				'bx--header__action--active': active
 			}"
-			[attr.aria-label]="title"
```

Comment:
Why does this need to be taken out?

Reply:
Header element with label as banner was reported as violation

Reply:
Facing the accessibility issue for that label 
Accessible name does not match or contain the visible label text

Reply:
Okay, that makes sense to me